lara-typescript/src/interactive-api-client/types.ts

```diff
@@ -349,6 +351,19 @@ export interface IAuthInfo {
   email?: string;
 }
 
+export interface IAttachmentUrlRequest extends IBaseRequestResponse {
+  name: string;
+  operation: "read" | "write";
```

Comment:
As mentioned above, it would be good to add a `timeout` or  `expires` field here. I'm not sure it is needed for writes because in those cases the client should write the content quickly and this could just be hardcoded to a few minutes. But for reads it is useful for the interactive to have some flexibility.